rllib/models/torch/primitives.py

```diff
@@ -0,0 +1,54 @@
+from typing import List, Optional
```

Comment:
Just some naming suggestions to keep in the back of our head for the upcoming catalog do-over. It'd be nice to use 3-letter shorts for all primitives:
* mlp
* cnn
* rnn
* (atn)?

Reply:
yep. nice suggestion. Don't know if we can keep this implicit convention going until we get a new layer (e.g. perceiver_io :) ?